### PR TITLE
Update install-kubeadm.md

### PR DESCRIPTION
**Pull request by kylejep**
Propose adding config reference to load br_netfilter at boot up to prevent issues with the module not being loaded after reboot.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@kylejep
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ Make sure that the `br_netfilter` module is loaded. This can be done by running
 As a requirement for your Linux Node's iptables to correctly see bridged traffic, you should ensure `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config, e.g.
 
 ```bash
+cat <<EOF | sudo tee /etc/modules-load.d/k8s.conf
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:62]: is this path consistent for different Linux distros?

**Comment by kylejep** [comment]
I signed it

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md (hunk 1 above)
Comment (kylejep) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:62]: My understanding is that any distro using systemd has service systemd-modules-load.service read config files from these paths at boot to determine which modules to load.
/etc/modules-load.d/*.conf
/run/modules-load.d/*.conf
/usr/lib/modules-load.d/*.conf

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:62]: ok, SGTM. thanks.

**Pull request merged**